awx/api/serializers.py

```diff
@@ -893,7 +893,7 @@ class Meta:
         model = User
         fields = ('*', '-name', '-description', '-modified',
```

Comment:
Kind of weird that `*` doesn't pick this up...

Reply:
Must have something to do w/ our usage of the base Django `auth_user` and the crazy metaclass that makes `Meta.fields` work.